frontend/packages/dev-console/src/components/formik-fields/multi-column-field/MultiColumnFieldRow.tsx

```diff
@@ -0,0 +1,34 @@
+import * as React from 'react';
+import { MinusCircleIcon } from '@patternfly/react-icons';
+
+export interface MultiColumnFieldRowProps {
+  name: string;
+  rowIndex: number;
+  children: React.ReactNode;
+  onDelete: () => void;
+}
+
+const MultiColumnFieldRow: React.FC<MultiColumnFieldRowProps> = ({
+  name,
+  rowIndex,
+  onDelete,
+  children,
+}) => (
+  <div className="odc-multi-column-field__row">
```

Comment:
Same comment about `className` belonging in its own file imported by this component file and following the naming conventions to align with the component name.

Reply:
Both `MultiColumnFieldRow` and `MultiColumnFieldHeader` are using the same css classes for styling rows and columns. That is why I've kept the styles in parent component.

Reply:
Please import the file where the classes are being used. Otherwise there's an implicit dependency on this file being imported by another component.

Reply:
Done